src/index.html

```diff
@@ -5,10 +5,26 @@
 		<title>CSV Search</title>
 		<meta content="initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no" name="viewport">
 		<link rel="shortcut icon" href="/static/app/favicon.ico" type="image/x-icon">
+		<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700' rel='stylesheet' type='text/css'>
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
I think we should bundle in as much as we can, because if the network is slow, more requests to different CDNs will make the problem worse, but I can move this later.